connect-button/publish.gradle

```diff
@@ -25,6 +25,7 @@ task sourcesJar(type: Jar) {
 }
 
 task javadoc(type: Javadoc) {
+    failOnError false
```

Comment:
Running into issues on this task: for some reason it could not generate BuildConfig correctly, and just for this module.